shared/cmd/flags.go

```diff
@@ -88,6 +88,13 @@ var (
 		Usage: "The IP address advertised by libp2p. This may be used to advertise an external IP.",
 		Value: "",
 	}
+	// P2PPrivKey defines a flag to specify the location of the private key file for libp2p.
+	P2PPrivKey = cli.StringFlag{
+		Name: "p2p-priv-key",
+		Usage: "The file containing the private key to use in communications with other peers. " +
+			"If unspecified a private key will be generated and saved to disk.",
```

Comment:
Lets stick with the current behavior of random peer identity. We don't need to persist identities at this time.

Reply:
I have removed the second line. Let me know if you want more color in there.